### PR TITLE
[operands] remove wasp-agent operand from HCO

### PR DESCRIPTION
**Pull request by enp0s3**
Signed-off-by: Igor Bezukh <ibezukh@redhat.com>

**What this PR does / why we need it**:
following the technical discussion we decided to deploy wasp agent in a different way, for the sake of easy transfer from wasp to kube swap.

```release-note
Remove wasp-agent from being deployed by HCO
```

### CODE REVIEW

File: pkg/components/components.go [hunk 1]
```diff
@@ -634,71 +629,6 @@ func GetClusterRoleBinding(namespace string) rbacv1.ClusterRoleBinding {
 	}
 }
 
-func GetWaspClusterRole() rbacv1.ClusterRole {
```
Comment (tiraboschi) [inline pkg/components/components.go:637]: please drop also "daemonsets" get/list/watch/create/update/delete

**Review by tiraboschi — CHANGES_REQUESTED** [review]

**Review by tiraboschi — APPROVED** [review]

**Comment by tiraboschi** [comment]
CI lanes for OKD are currently broken
/override ci/prow/okd-hco-e2e-upgrade-operator-sdk-aws
/override ci/prow/okd-hco-e2e-upgrade-operator-sdk-gcp

**Comment by stesrn** [comment]
Verification of this patch is successful - https://main-jenkins-csb-cnvqe.apps.ocp-c1.prod.psi.redhat.com/job/openshift-virtualization-tests-runner/55/testReport/

**Pull request merged**